src/certs/sev/builtin/mod.rs

```diff
@@ -4,9 +4,24 @@
 //!
 //! These are primarily offered as a convenience measure to avoid making
 //! HTTP requests to AMD's servers.
-
```

Comment:
You left a `deprecated` note on this module in `src/certs/mod.rs`. Why are these deprecation notes also needed?